### PR TITLE
Make account names unique within a bookset

### PR DESCRIPTION
**Pull request by omarkhan**
The bookset api uses `.get()` to fetch accounts by name. This implies that account names should be unique within a bookset, as `.get()` will raise an exception if there is more than one result.

This pull request makes this explicit by preventing multiple accounts with the same name in the same bookset.

### CODE REVIEW

File: swingtix/bookkeeper/test_model.py [hunk 1]
```diff
@@ -352,7 +350,6 @@ def test_book_set_basic(self):
             description="Expenses",
             positive_credit=False
         )
-        expense.save()
```
Comment (g--) [inline swingtix/bookkeeper/test_model.py]: Hi Omar!  Thanks for your contributions to bookkeeper!

I don't understand why .save() needs to be removed here.. I must be missing something?

Comment (omarkhan) [inline swingtix/bookkeeper/test_model.py]: Sorry should have put that in a separate PR. `save()` is redundant here as `Account.objects.create(...)` already saves the object: https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.create

Comment (g--) [inline swingtix/bookkeeper/test_model.py]: TIL!  thanks for that!
